libs/wingc/src/lib.rs

```diff
@@ -30,6 +30,10 @@ pub mod utilities;
 const WINGSDK_ASSEMBLY_NAME: &'static str = "@winglang/wingsdk";
 
 const WINGSDK_DURATION: &'static str = "std.Duration";
+const WINGSDK_ARRAY: &'static str = "std.ImmutableArray";
+// const WINGSDK_MUTARRAY: &'static str = "std.MutableArray";
```

Comment:
?

Reply:
Adding commented code is a sin indeed, will remove